docs/src/development/local/_index.md

```diff
@@ -87,13 +85,16 @@ Your environments are:
 
 ### Launch the download
 
-To download a copy of your app's code, run the following command. 
-{{< variable "DIRECTORY_NAME" >}} is the name you want to give to the target project directory on your computer:
+If your app's code isn't already present locally on your computer,
+you can download a copy of it.
+To do so, move to the directory where you want to save a copy of your project 
+and run this command:
 
 ```bash
-platform get {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT_NAME" >}} {{< variable "DIRECTORY_NAME" >}}
+platform get {{< variable "PROJECT_ID" >}} --environment {{< variable "ENVIRONMENT_NAME" >}}
 ```
 
+When prompted, choose a name for the target project directory and press **Enter**.
 You get output similar to the following:
 
 ```
```

Comment:
:lady_beetle:
it should be `project-name` instead of `project-id`.

```
Downloading [project-name](abcdefgh1234567)
```

Also, above [the region should be `eu`.](https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#use-clear-examples)